cmake/modules/hunter_fatal_error.cmake

```diff
@@ -8,7 +8,7 @@ include(hunter_wiki)
 
 function(hunter_fatal_error)
   cmake_parse_arguments(hunter "" "WIKI" "" "${ARGV}")
-  if(NOT hunter_WIKI)
```

Comment:
It make sense to use `x_` here instead of `hunter_`:
```cmake
cmake_parse_arguments(x "" "WIKI" "" "${ARGV}")
if(NOT x_WIKI)
  hunter_internal_error("Expected wiki")
endif()
```